requirements.txt

```diff
@@ -2,3 +2,5 @@ six==1.11.0
 pandas==0.22.0
 numpy==1.14.0
 typing==3.6.4
+# for the tool
+tensorflow==1.8.0
```

Comment:
This comes from https://github.com/spotify/spotify-tensorflow/blob/master/setup.cfg#L28-L30 do we need an explicit dependency on TF for the whole of SPTF?

Reply:
agree, I'd like to avoid that